base/db/R/convert.input.R

```diff
@@ -736,7 +737,7 @@ convert.input <-
     log_format_df = function(df){
       df  %>%
         format() %>%
-        rbind(colnames(.), .) %>%
+        rbind(colnames(format(df)), format(df))
         purrr::reduce( paste, sep=" ") %>%
         paste(collapse="\n")
     }
```

Comment:
If you do this, you need to get rid of `df %>%` as well.  Please test this function on any dataframe before asking for review.